uftrace.h

```diff
@@ -270,6 +270,8 @@ struct opts {
 	enum uftrace_pattern_type patt_type;
 };
 
+extern struct strv uftrace_opts;
```

Comment:
I think `replay_opt_str` or `analysis_opt_str` is better.

Reply:
I've used `uftrace_opts` in many places including the related functions. If it's changed, maybe we better change all the other places. And I personally think the current name is better.

I can rather change the name `uftrace.opts` itself to something else if there's better one.

Reply:
`default.opts`?

Reply:
It looks better. I will change all the namings to that. Thanks.

Reply:
I changed it to `default.opts`.